contracts/VestaParameters.sol

```diff
@@ -28,6 +29,8 @@ contract VestaParameters is IVestaParameters, OwnableUpgradeable, CheckContract
 	mapping(address => uint256) public override MCR;
 	// Critical system collateral ratio. If the system's total collateral ratio (TCR) falls below the CCR, Recovery Mode is triggered.
 	mapping(address => uint256) public override CCR;
+	// Bonus for individual troves
+	mapping(address => uint256) public override BONUS;
```

Comment:
BONUS isn't a good name. Bonus of what? bonus for what?

something like: LIQUIDATION_PAY_BACK_PERCENT will be better. (Or anything else if you have ideas)